esmvaltool/cmorizers/data/formatters/datasets/ct2019.py

```diff
@@ -182,10 +182,9 @@ def _extract_variable(short_name, var, cfg, input_files, out_dir):
                         unlimited_dimensions=['time'])
 
 
-def cmorization(in_dir, out_dir, cfg, _):
+def cmorization(in_dir, out_dir, cfg, _, start_date, end_date):
```

Comment:
Just wondering: in this script `start_date` and `end_date` are used in the call to the `cmorization` function while `start` and `end` are used in others. Which one is correct?